<launch>
  <arg name="veh" doc="Name of vehicle. ex: megaman"/>
  <arg name="package_namespace" default="calibration" />
  <arg name="node_namespace" default="optical_flow" />

  <!-- Launch the script that generates optical flow grid -->
  <group ns="$(arg veh)/$(arg package_namespace)">
    <node  pkg="calibration" type="$(arg node_namespace).py" name="$(arg node_namespace)" output="screen" clear_params="true" required="true">
      <param name="veh" value="$(arg veh)"/>
    </node>
  </group>
</launch>
